src/Microbuild.Settings.targets

```diff
@@ -0,0 +1,205 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
This file doesn't look like it is part of F# itself, but rather part of MicroBuild.
It feels like it should come from the MicroBuld nuget package, or be in some path like ``.microbuild\Microbuild.Setting.targets``, rather than in ``src``